Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -464,6 +464,36 @@ public extension UIView {
 		}
 	}
 
+    /// SwifterSwift: Anchors all sides to superview's safe area.
+    ///
+    /// 1. On iOS 11 and latter anchors view's sides to superview's safe area
+    /// 2. On iOS 9-11 anchors leading and trailing to superview,
+    /// top and bottom to viewController's bottomLayoutGuide and topLayoutGuide.
+    ///
+    /// - Parameter viewController: view controller, of which bottomLahoutGuide
+    ///  and topLayoutGuide are used on iOS 9-11
+    @available(iOS 9, *) public func fillToSuperviewSafeArea(viewController: UIViewController) {
```

Comment:
I'm not sure if the parameter the view controller here is necessary. 
Can you explain a use case where passing the view controller just to use the topLayoutGuide necessary?
And not just use superview. topAnchor is enough.

Reply:
At first I wanted to pass `bottomLayoutGuide` and `topLayoutGuide` as parameters, but if you use them in your code, you get deprecation warning. However, in the function `bottomLayoutGuide` and `topLayoutGuide` are used inside `else` closure to `#available(iOS 11, *)` so the compiler doesn't generate deprecation warning.